package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@prezly/sdk",
-    "version": "2.1.0",
+    "version": "2.1.1",
```

Comment:
For the record: this is done automatically by `np` command.

All you had to do is to run `np patch` -- it'd do the rest, including bumping this version and publishing the package to `npm`.

Reply:
I would looooooove to [automate this](https://discourse.prezly.io/t/versioning-in-ci/160). 🙂

Reply:
File a PR :)